interface/usergroup/addrbook_edit.php

```diff
@@ -528,5 +533,41 @@ function typeSelect(a) {
 
 </center>
 </form>
+
+<script>
```

Comment:
It would indeed be nice to look for an email field function already existing somewhere...but actually, I recommend just using HTML5 input type "email".  That takes care of email validation all on it's own, unless they are using IE9 or earlier.